Microsoft.Toolkit.Uwp.UI.Controls/ImageEx/RoundImageEx.xaml

```diff
@@ -0,0 +1,111 @@
+﻿<ResourceDictionary xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+                    xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+                    xmlns:controls="using:Microsoft.Toolkit.Uwp.UI.Controls">
+
+    <Style TargetType="controls:RoundImageEx">
+        <Setter Property="Background" Value="Transparent" />
+        <Setter Property="Foreground" Value="{ThemeResource ApplicationForegroundThemeBrush}" />
+        <Setter Property="Template">
+            <Setter.Value>
+                <ControlTemplate TargetType="controls:RoundImageEx">
+                    <Grid>
+                        <Ellipse x:Name="PlaceholderEllipse"
```

Comment:
The Control name wouldn't make sense anymore. I could name it ImageBrushEx, although it would still be encapsulated so it wouldn't really be an ImageBrush you could attach to things,